hw/top_darjeeling/lint/top_darjeeling.waiver

```diff
@@ -36,6 +36,14 @@ waive -rules {HIER_BRANCH_NOT_READ} -location {pinmux.sv} \
 waive -rules INPUT_NOT_READ -location {pinmux.sv} -regexp {Input port '(rst_sys_ni|scanmode_i)' is not read from} \
       -comment "This signal is unused but cannot yet be gated in ipgen."
 
+waive -rules {HIER_NET_NOT_READ NOT_READ} -location {clkmgr.sv} \
+      -regexp {(Net|Signal) 'clk_io_root' is not read from in module 'clkmgr'} \
+      -comment "This clock is unused in Darjeeling - should not be generated"
+
+waive -rules {HIER_NET_NOT_READ NOT_READ} -location {clkmgr.sv} \
+      -regexp {(Net|Signal) 'clk_io_div2_root' is not read from in module 'clkmgr'} \
+      -comment "This clock is unused in Darjeeling  - should not be generated"
+
```

Comment:
I've filed #25987 to keep track of not generating these clocks. Should be possible but not a priority right now.